clients/src/main/java/org/apache/kafka/common/utils/Java.java

```diff
@@ -41,4 +41,8 @@ private Java() {
     public static final boolean IS_JAVA9_COMPATIBLE = JVM_MAJOR_VERSION > 1 ||
             (JVM_MAJOR_VERSION == 1 && JVM_MINOR_VERSION >= 9);
 
+    public static boolean isIBMJdk() {
+        return System.getProperty("java.vendor").contains("IBM");
+    }
```

Comment:
We use the same IBM JDK check in `SaslChannelBuilder` and `MiniKdc`. Perhaps those could be changed to use `Java.isIBMJdk` as well? Doesn't have to be in this PR.